var/spack/repos/builtin/packages/eigen/package.py

```diff
@@ -39,9 +40,15 @@ class Eigen(CMakePackage, ROCmPackage):
     version("3.2.5", sha256="8068bd528a2ff3885eb55225c27237cf5cda834355599f05c2c85345db8338b4")
 
     variant("nightly", description="run Nightly test", default=False)
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+
+    # TODO: https://eigen.tuxfamily.org/dox/TopicUsingBlasLapack.html
+
+    # Using fortran is optional up to v3.4.0, can be disabled in develop see EIGEN_BUILD_BLAS
+    # depends_on("fortran", type="build")  # generated
```

Comment:
I think the wording here can be a bit clearer; as I understand the Eigen documentation, it is perfectly valid to have no Fortran throughout and link to say, the MKL. The fortran dependency is anyway not explicit (should be implied by linkage to a `blas/lapack`).

Reply:
@HaoZeke The comments at ilne 47, and the at line 49 are separate. For the latter I was referring to this https://gitlab.com/libeigen/eigen/-/blob/master/blas/CMakeLists.txt?ref_type=heads

It seems to me Fortran is necessary when setting `EIGEN_BUILD_BLAS`, which seems to be a separate, and optional, CMake project.

Reply:
Thanks for clearing that up, does it make sense to remove that portion (L49-50) completely then? Since it seems like `eigen-blas` would be a separate package with `eigen` as a dependency..

Reply:
That's fine with me, let me do it.

Reply:
```suggestion
    # TODO: https://eigen.tuxfamily.org/dox/TopicUsingBlasLapack.html
```